tools/athena-start

```diff
@@ -0,0 +1,227 @@
+#!/bin/sh
```

Comment:
Can we rename it to .sh while we are at it?
And shouldnt the scripts you moved into tools be removed from the root then?

Reply:
I don't want to break the current workflow for now, but I put a log in the old `athena-start` script telling people to use cmake and `./athena-start.sh` instead